ipatests/test_integration/test_krbtpolicy.py

```diff
@@ -176,4 +183,27 @@ def test_krbtpolicy_otp(self):
             del_otptoken(master, otpuid)
             reset_to_default_policy(master, USER1)
             self.master.run_command(['rm', '-f', armor])
-            master.run_command(['ipa', 'config-mod', '--user-auth-type='])
+
+    def test_krbtpolicy_jitter(self, reset_policy):
+        """Test jitter lifetime with no auth indicators"""
+        tasks.kinit_admin(self.master)
+        kinit_check_life(self.master, USER1)
+
+    def test_krbtpolicy_jitter_otp(self, reset_policy):
+        """Test jitter lifetime with OTP"""
+        tasks.kinit_admin(self.master)
+        self.master.run_command(["ipa", "user-mod", USER1,
+                                 "--user-auth-type", "otp"])
+        self.master.run_command(["ipa", "config-mod",
+                                 "--user-auth-type", "otp"])
+        kinit_check_life(self.master, USER1)
+
+    def test_krbtpolicy_jitter_pkinit(self, reset_policy):
+        """Test jitter lifetime with PKINIT"""
+        tasks.kinit_admin(self.master)
+        self.master.run_command(["ipa", "user-mod", USER2,
+                                 "--user-auth-type", "pkinit"])
+        self.master.run_command(["ipa", "config-mod",
+                                 "--user-auth-type", "pkinit"])
+        kinit_check_life(self.master, USER2)
+        reset_to_default_policy(self.master, USER2)
```

Comment:
In test_krbtpolicy_default() and test_krbtpolicy_jitter() the global auth type config isn't being modified so strictly speaking you don't need reset_policy as a fixture.

Is changing botht eh user and the global auth type necessary to check the jitter?